build/teamcity-roachtest-stress.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+generate_ssh_key
+log_into_gcloud
+export ROACHPROD_USER=teamcity
+
+build/builder/mkrelease.sh amd64-linux-gnu build bin/workload bin/roachtest bin/roachprod 
+
+build/teamcity-roachtest-invoke.sh \
```

Comment:
How is this preventing issues from getting posted for test failures?

Reply:
I don't think it is, will look into it.

Reply:
Added a commit with a `roachtest run --disable-issue` flag.